procedures/foreman/branch.md.erb

```diff
@@ -74,10 +68,12 @@
 
 ## Release Engineer
 
-- Branch RPM packaging using [tool_belt](https://github.com/theforeman/tool_belt)
-  - [ ] Clone tags and create build targets in Koji: `bundle exec ./tools.rb koji --commit configs/foreman/<%= release %>.yaml`
-    Note that you must use koji client < 1.30 since clone-tag was changed and our Koji server is too old. See https://docs.pagure.org/koji/release_notes/release_notes_1.30/#system-changes for details.
-  - [ ] Create a `rpm/<%= release %>` branch in [foreman-packaging](https://github.com/theforeman/foreman-packaging) based on `rpm/develop`
+- Branch RPM packaging
+  - [ ] Create a `rpm/<%= release %>` branch in [foreman-packaging](https://github.com/theforeman/foreman-packaging) based on `rpm/develop`: `git checkout rpm/develop && git pull && git checkout -b rpm/<%= release %>`
+  - [ ] Update `foreman_version` in `package_manifest.yaml` on the `rpm/<%= release %>` branch: `sed -i '/foreman_version:/ s/nightly/<%= release %>/' package_manifest.yaml`
+  - [ ] Update `katello_version` in `package_manifest.yaml` on the `rpm/<%= release %>` branch: `sed -i '/katello_version:/ s/nightly/KATELLO_VERSION_HERE/' package_manifest.yaml
+  - [ ] Create release repositories in Copr by forking nightly repositories `obal copr-project copr_projects` on the `rpm/<%= release %>` branch
```

Comment:
This will require the release engineer to have a working copr cli setup and the right permissions.
I do not recall if we document this properly today for koji (probably not), but worth fixing this now :)

Reply:
We do not have this documented... should it be here? Should it be on foreman-packaging (pulpcore-packaging, etc?)? Somewhere else more central?

Reply:
Added